.gitignore

```diff
@@ -37,3 +37,4 @@ coverage
 # Structurizr
 .structurizr
 workspace.json
+pyrightconfig.json
```

Comment:
out of interest, how have you configured pyright?

Reply:
very simple setup, but neovim wasn't happy without:

```
{
  "venvPath": ".",
  "venv": ".venv",
  "include": [
    "manage_breast_screening"
  ]
}
```